rsc/log4j.properties

```diff
@@ -12,28 +12,29 @@ log4j.appender.StdOut.layout.ConversionPattern=[%d{dd/MMM/yyyy HH:mm:ss}] [%X{OH
 # File Appender (with classes), daily rotation
 log4j.appender.RollingFile = org.apache.log4j.DailyRollingFileAppender
 log4j.appender.RollingFile.DatePattern='.'yyyy-MM-dd
-log4j.appender.RollingFile.File=LOG_DEST
+log4j.appender.RollingFile.File=/home/andrei/oh
 log4j.appender.RollingFile.layout=org.apache.log4j.PatternLayout
 log4j.appender.RollingFile.layout.ConversionPattern=[%d{dd/MMM/yyyy HH:mm:ss}] [%X{OHUserGroup}:%X{OHUser}] %-p - %m (%l)%n
 
 # DB Appender (table columns)
 log4j.appender.DB=org.apache.log4j.jdbc.JDBCAppender
-log4j.appender.DB.URL=jdbc:mysql://DBSERVER:DBPORT/DBNAME?autoReconnect=true
-log4j.appender.DB.user=DBUSER
-log4j.appender.DB.password=DBPASS
+log4j.appender.DB.URL=jdbc:mysql://localhost:3307/oh?autoReconnect=true
```

Comment:
hard coding values in this file appears to be wrong as the install sets the values.